cpp/simulations/abm.cpp

```diff
@@ -849,9 +846,14 @@ int main()
     mio::abm::close_social_events(t_lockdown, 0.9, world.get_migration_parameters());
 
     auto sim = mio::abm::Simulation(t0, std::move(world));
+    return sim;
+}
 
-    sim.advance(tmax);
-
+/**
+ * Save the current run result to a directory.
+ * @param results the result in form of a TimeSeries object of infection_state individuals for the whole world
+*/
+void save_result(mio::TimeSeries<double> results, size_t run_idx, const fs::path& result_dir) {
```

Comment:
Why do you hand over a copy of the TimeSeries object? Big objects should be handed over as references and here in particular as constant ref.

Reply:
Errors on my side. It's now fixed!